anchore_engine/services/policy_engine/api/controllers/synchronous_operations.py

```diff
@@ -998,8 +997,21 @@ def get_image_vulnerabilities(user_id, image_id, force_refresh=False, vendor_onl
                             api_endpoint, vulnerability_cpe.vulnerability_id
                         )
 
+                    nvd_data = vulnerability_cpe.parent.get_cvss_data_nvd()
+                    vulndb_id = vulnerability_cpe.vulnerability_id
+
+                    # Check to see if a single CVE id is located in the nvd data
+                    # and present it to allow for easy cross reference from 
+                    # other sources and use public links for vulnerability information.
+                    # issue #442
+
+                    if vulndb_id and len(nvd_data) == 1:
```

Comment:
This condition is a little broad and and results true for nvd vulnerabilities too which is not necessary. I think a check like this, if necessary, should be over the type `isInstance(vulnerability_cpe, VulnDBCpe)` or the feed `vulnerability_cpe.feed == "vulndb"`

Reply:
Ok, thank you for the feedback.